qa-brain.planx-pla.net/portal/gitops.json

```diff
@@ -231,8 +231,20 @@
         {
           "title": "Assessment",
           "fields": [
+            "unified_parkinsons_disease_ratings",
             "mds_updrs",
-            "mds_updrs_part_iii_summary_score"
+            "hopkins_verbal_learning_tests",
+            "scales_for_outcomes_in_pds",
```

Comment:
do you want to put this into `fieldMapping`? Since `pds` loks like an abbreviation

Reply:
I'd like to see how it looks as is, and then change it later